File: script/EFIT/sensA2.py
import random

def sens_a2():
    opts = {
        'thr': 0.1,
        'F0iidtol': 0.005,
        'maxFunEvals': 25000,
        'maxIter': 25000,
        'verbose': False,
    }

    Tmin = 10
    Tmax = 60

    Ts = round(random.uniform(Tmin, Tmax))

    return Ts

result = sens_a2()
print(result)

File: script/EFIT/Stores/EMOTpackage/private/sensA2.m
function [] = sensA2()

opts.thr=0.1;
opts.F0iidtol = 0.005;
opts.maxFunEvals = 25000;
opts.maxIter = 25000;
opts.verbose = false;

Tmin=10; Tmax=60;

Ts = round(random('uniform',Tmin,Tmax,1,1))


end